float32 energy_aV
float32 water_aV
float32 temp_aV
float32 hunger_err
float32 thirst_err
float32 temp_err
float32 adsign
float32 hsign
